src/nyc_trees/apps/users/models.py

```diff
@@ -114,10 +111,9 @@ def __init__(self, name, description, description_achieved, badge,
         badge='img/badges/ic_badge_map_200.png',
         achieved=lambda user: user.blocks_mapped_count >= 200,
         sponsor='Sponsored by Whole Foods Market',
-        reward='TreesCount! Tote &amp; Tree Care Kit '
-               'that includes gloves, weeder, '
-               'cultivator, trowel, a collapsible bucket '
-               'and other helpful tools'
+        reward='TreesCount! Tote & Tree Care Kit that includes a limited '
```

Comment:
If we keep the `|safe` template filter in place, this `&` will need to stay as `&amp;`.  If we do remove the `|safe`, I _think_ it will be ok to leave it as a literal `&`, but we should double check.